docs/UserGuide.md

```diff
@@ -242,26 +242,45 @@ to his Travel Insurance plan that has a INSURANCE_ID of 1.
 
 ### Closing a claim of a client : `closeClaim`
 
-Closes a claim from an insurance plan of a client. The claim will still be tagged to the client
-in the system but marked as closed.
+Closes a claim from an insurance plan of a client.
 
 Format: `closeClaim INDEX iid/INSURANCE_ID cid/CLAIM_ID`
 
 IDs for insurance plans:
 `Basic Insurance Plan` : `0`,
 `Travel Insurance Plan`: `1`
 
-* Closes a claim from the client at the specified `INDEX`.
-* `INSURANCE_ID` are found in the address book matching the associated insurance plan.
+<box type="tip" seamless>
+    Parameters
+</box>
+
+1. `INDEX` : Must be a valid client index in the filtered list that is currently shown.
+   <br>
+2. `INSURANCE_ID` : Must be an existing valid ID that is currently supported.
+   <br> Currently supported IDs for insurance plans: `Basic Insurance Plan` : `0` & `Travel Insurance Plan`: `1`. <br>
+3. `CLAIM_ID` : a unique id for each claim generated by the company and should be created externally by another
+   organisation before the claim is added to this app.
```

Comment:
what about replacing `another organisation` with `the insurance company` to make it clearer 🙂

Reply:
cause can be some other law office also, but can change that to external organisation if u want

Reply:
Will do :)